DECISIONS.md

```diff
@@ -0,0 +1,119 @@
+# Decisions
+
+This document is intended to provide clarity on many of the decisions/rationalizations
+which exist inside of [Datamill's](https://datamill.co/) Target SQL project
+for [Singer](https://singer.io).
+
+The guiding principles we try to adhere to herein as far as _how_ to reach a
+conclusion are:
+
+1. When possible, make the resulting data/schema in the remote target consistent, no matter the ordering of potential messages
+  - ie, if our decision would result in a random schema being produced in the remote target for no reasonable benefit, this is in violation
+1. Do right by the common _majority_ of users
+1. Make a best effort to prevent a user from having to intervene
+1. Use [Stitch’s offering and documentation](https://www.stitchdata.com/docs) as best practice guidance
+
+## Schema
+
+### De-nesting
+
+#### What
+
+- [JSON Schema](https://json-schema.org/) allows for complex schemas which have non-literal (ie, compositional) elements
```

Comment:
Maybe define `non-literal (ie, compositional) elements` more and give examples? Same may call them basic types, scalar values, etc.